examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,30 +62,29 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-clf = make_pipeline(Xdawn(n_components=3),
-                    EpochsVectorizer(),
+X = EpochsVectorizer().fit_transform(epochs)
+labels = epochs.events[:, -1]
+clf = make_pipeline(XdawnTransformer(n_components=3,
+                    n_chan=epochs.info['nchan']),
                     MinMaxScaler(),
                     LogisticRegression(penalty='l1'))
 
-# Get the labels
-labels = epochs.events[:, -1]
-
 # Cross validator
 cv = StratifiedKFold(y=labels, n_folds=10, shuffle=True, random_state=42)
```

Comment:
To keep the example to the strict minimum, I would replace the cv and following report to just
`score = cross_val_scores(X, y)`
`print(score)`